### PR TITLE
Feature/missing envelop icon 432

### PR DESCRIPTION
**Pull request by omarzoghayyer**
### What does this pull request aim to achieve? 💯

****Added missing envelope icon on the header section. This also will be linked to the feed page.****

Please ignore the files being replaced. I ended up coping and paste the whole page which results in this formate. 

### 🚨Before submitting this pull request🚨:

_Please do **NOT** submit this PR if you have not done the following:_

1. I have checked that no one else is working on similar changes.
2. I have read the latest [**README**](https://github.com/FightPandemics/FightPandemics/blob/master/README.md) and understand the development workflow.
3. The name of this branch is: **`feature/<branch_name>`** (need to have cloned the repo not forked).
4. This branch is rebased/merged with the **latest master**.
5. There are no merge conflicts.
6. There are no console warnings and errors.
7. On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".

### CODE REVIEW

**Comment by robinv85** [comment]
@omarzoghayyer I thought the envelope was meant for the user feedback functionality? That is still a work in progress.
But I do find it confusing, the envelope isn't clear to me that it should be about giving feedback.

**Comment by omarzoghayyer** [comment]
@robinv85  the envelop should direct to the following scree below, and as you said they are still in progress. We can easily change the directory once they're done. 
Screen Shot 2020-05-26 at 4 24 53 PM

File: client/src/components/Header.js [hunk 1]
```diff
@@ -3,102 +3,92 @@ import { NavBar } from "antd-mobile";
 import { Link, NavLink } from "react-router-dom";
 // import { Menu as MenuIcon } from "grommet-icons";
 import styled from "styled-components";
-
+import envelope from "assets/icons/envelope.svg";
```
Comment (silenceway) [inline client/src/components/Header.js:6]: Move this line to the icons group.

File: client/src/components/Header.js [hunk 2]
```diff
@@ -3,102 +3,92 @@ import { NavBar } from "antd-mobile";
...
-  height: 7rem;
-  margin-top: 0;
-  @media screen and (max-width: ${mq.phone.wide.maxWidth}) {
-    height: auto;
-    margin-top: 0.8rem;
-  }
-  .am-navbar-title {
-    display: none;
-  }
+   height: 7rem;
```
Comment (silenceway) [inline client/src/components/Header.js:22]: Check this indentation, it's not 2 spaces.

File: client/src/components/Header.js [hunk 3]
```diff
@@ -3,102 +3,92 @@ import { NavBar } from "antd-mobile";
...
-
 const DesktopMenu = styled.div`
-  background-color: #fff;
-  display: flex;
-  align-items: center;
-  display: block;
-  @media screen and (max-width: ${mq.phone.wide.maxWidth}) {
-    display: none;
-  }
+   background-color: #fff;
```
Comment (silenceway) [inline client/src/components/Header.js:40]: Check the indentation.

File: client/src/components/Header.js [hunk 4]
```diff
@@ -3,102 +3,92 @@ import { NavBar } from "antd-mobile";
...
-        border-bottom: 3px solid transparent;
-      }
-      a:hover:not(.registerLink) {
-        font-weight: 600;
-        color: ${colors.royalBlue};
-        border-bottom: 3px solid ${colors.royalBlue};
-      }
-    }
-  }
+   ul {
```
Comment (silenceway) [inline client/src/components/Header.js:51]: Check indentation.

File: client/src/components/Header.js [hunk 5]
```diff
@@ -116,47 +106,31 @@ export default ({ onMenuClick, ...props }) => {
...
-                      <li className="registerBtn">
-                        <NavLink className="registerLink" to="/auth/signup">
-                          Register
-                        </NavLink>
-                      </li>
-                    </>
-                  )}
-                </ul>
-              </NavLinks>
+          <DesktopMenu>
```
Comment (silenceway) [inline client/src/components/Header.js:109]: Check indentation.

File: client/src/components/Header.js (hunk 3 above)
Comment (silenceway) [reply client/src/components/Header.js:40]: Color should be set from themes

File: client/src/components/Header.js [hunk 6]
```diff
@@ -3,102 +3,92 @@ import { NavBar } from "antd-mobile";
...
+     }
+   }
 `;
-
 const activeStyles = {
   fontWeight: "600",
-  color: "#425AF2",
-};
-
+  color: "#425AF2"
```
Comment (silenceway) [inline client/src/components/Header.js:90]: Color should be set from themes.

File: client/src/components/Header.js [hunk 7]
```diff
@@ -3,102 +3,92 @@ import { NavBar } from "antd-mobile";
...
+       .registerLink {
+          color: ${colors.primary}
+       }
+       .registerLink:hover {
+         font-weight: 500;
+       }
+     }
+     li {
+       font-size: ${large};
+       color: #282828;
```
Comment (silenceway) [inline client/src/components/Header.js:72]: Color should be set from themes.

File: client/src/components/Header.js [hunk 8]
```diff
@@ -3,102 +3,92 @@ import { NavBar } from "antd-mobile";
...
+       a:not(.registerLink) {
+         color: #282828;
```
Comment (silenceway) [inline client/src/components/Header.js:74]: Color should be set from themes.

File: client/src/components/Header.js [hunk 9]
```diff
@@ -3,102 +3,92 @@ import { NavBar } from "antd-mobile";
...
-  display: flex;
-  align-items: center;
-  display: block;
-  @media screen and (max-width: ${mq.phone.wide.maxWidth}) {
-    display: none;
-  }
+   background-color: #fff;
+   display: flex;
+   align-items: center;
+   display: block;
```
Comment (silenceway) [inline client/src/components/Header.js:43]: Why display flex and then display block. Clean the rule.

**Comment by omarzoghayyer** [comment]
@silenceway Thank you for pointing the silly mistakes. They should be done now. Please let me know if there is anything else I can improve.

File: client/src/components/Header.js [hunk 10]
```diff
@@ -3,54 +3,47 @@ import { NavBar } from "antd-mobile";
...
-  height: 7rem;
-  margin-top: 0;
-  @media screen and (max-width: ${mq.phone.wide.maxWidth}) {
-    height: auto;
-    margin-top: 0.8rem;
-  }
-  .am-navbar-title {
-    display: none;
-  }
+   height: 7rem;
```
Comment (silenceway) [inline client/src/components/Header.js:22]: Still seeing 3 spaces instead of 2.

File: client/src/components/Header.js [hunk 11]
```diff
@@ -3,54 +3,47 @@ import { NavBar } from "antd-mobile";
...
+     display: none;
+   }
 `;
-
 const MenuToggle = styled(SvgIcon)`
   cursor: pointer;
   display: none !important;
   @media screen and (max-width: ${mq.phone.wide.maxWidth}) {
-    display: block !important;
+  display: block !important;
```
Comment (silenceway) [inline client/src/components/Header.js:36]: Wrong indentation. Should be 2 spaces.

File: client/src/components/Header.js [hunk 12]
```diff
@@ -3,54 +3,47 @@ import { NavBar } from "antd-mobile";
...
-
 const DesktopMenu = styled.div`
-  background-color: #fff;
+  background-color: ${colors.white};
   display: flex;
   align-items: center;
-  display: block;
   @media screen and (max-width: ${mq.phone.wide.maxWidth}) {
-    display: none;
+  display: none;
```
Comment (silenceway) [inline client/src/components/Header.js:44]: Wrong indentation. Should be 2 spaces.

File: client/src/components/Header.js [hunk 13]
```diff
@@ -59,104 +52,84 @@ const NavLinks = styled.div`
...
-        border-bottom: 3px solid transparent;
-      }
-      a:hover:not(.registerLink) {
-        font-weight: 600;
-        color: ${colors.royalBlue};
-        border-bottom: 3px solid ${colors.royalBlue};
-      }
-    }
-  }
+     .registerBtn {
```
Comment (silenceway) [inline client/src/components/Header.js:55]: Wrong indentation. Should be 4 spaces, not 6.

File: client/src/components/Header.js [hunk 14]
```diff
@@ -59,104 +52,84 @@ const NavLinks = styled.div`
...
+       .registerLink {
+         color: ${colors.primary}
+        }
+       .registerLink:hover {
+         font-weight: 500;
+       }
+     }
+     li {
+       font-size: ${large};
+       color: ${colors.DARKER_GRAY};
```
Comment (silenceway) [inline client/src/components/Header.js:71]: colors.darkerGray

File: client/src/components/Header.js [hunk 15]
```diff
@@ -59,104 +52,84 @@ const NavLinks = styled.div`
...
+       a:not(.registerLink) {
+         color: ${colors.DARKER_GRAY};
```
Comment (silenceway) [inline client/src/components/Header.js:73]: colors.darkerGray

File: client/src/components/Header.js [hunk 16]
```diff
@@ -59,104 +52,84 @@ const NavLinks = styled.div`
...
+}
 export default ({ onMenuClick, ...props }) => {
   return (
     <div className="header" {...props}>
       <StyledNavBar
-        mode="light"
-        leftContent={
-          <BrandLink to="/">
-            <Logo src={logo} alt="Fight Pandemics logo" />
+      mode="light"
```
Comment (silenceway) [inline client/src/components/Header.js:95]: Wrong indentation.

**Comment by robinv85** [comment]
@omarzoghayyer ah ok... well then I guess the pop-up could be done with the query parameter instead of a redirect, similar to "create-post" modal so it can open from witthin different "pages" or "views" and it doesn't have its own route

**Comment by omarzoghayyer** [comment]
@silenceway can you check them now. I wonder whats wrong with my VScode.

**Comment by silenceway** [comment]
> @silenceway can you check them now. I wonder whats wrong with my VScode.

Sorry, it's not fixed. Header.js line 24 (@media screen) has a green block showing a new space that is wrong. Line 62 (.registerLink) has 5 spaces. Line 95 (mode=) misses one space (pink block). Please review.

**Comment by robinv85** [comment]
@omarzoghayyer do you have ESLint & Prettier configured correctly?
Also, you need to set the .env for ESLint custom rules...

**Comment by omarzoghayyer** [comment]
@robinv85 Yes I have ESLint configured. As for Prettier. Every time I enable Format on Save. It messes up the whole code structure, and I get so mush errors. Not sure why is this happening. 
Screen Shot 2020-05-27 at 3 00 32 PM

**Comment by omarzoghayyer** [comment]
@silenceway  I think it should be good to go now. Could you please check? Thank you.

File: client/src/components/Header.js [hunk 17]
```diff
@@ -74,31 +66,28 @@ const NavLinks = styled.div`
...
       a:not(.registerLink) {
-        color: #282828;
+        color: ${colors.darkerGray};
         text-decoration: none;
         padding: 1.2rem 1.4rem;
         transition: all 0.2s;
         border-bottom: 3px solid transparent;
       }
-      a:hover:not(.registerLink) {
+      a:hov er:not(.registerLink) {
```
[the remark this answers is not in the document]
Comment (omarzoghayyer) [reply client/src/components/Header.js:79]: on thats bad. One second.

**Pull request merged**